### PR TITLE
render xml properly in the grid

### PR DESCRIPTION
**Pull request by alanrenmsft**
#4648

Root cause: we escaped the strings in profiler as a result the xml cannot be displayed properly.
Fix: remove the escape call in profiler, this is safe since the underlying slickgrid already have the escaping logic.

### CODE REVIEW

**Comment by alanrenmsft** [comment]
image

File: src/sql/parts/profiler/editor/profilerInput.ts [hunk 1]
```diff
@@ -258,7 +258,7 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 				let columnName = this._columnMapping[key];
 				if (columnName) {
 					let value = e.values[key];
-					data[columnName] = escape(value);
+					data[columnName] = value;
```
Comment (Charles-Gagnon) [inline src/sql/parts/profiler/editor/profilerInput.ts]: Might be worth it to document this so we don't try to "fix" it again in the future.

Comment (Charles-Gagnon) [reply src/sql/parts/profiler/editor/profilerInput.ts]: Oh and you can remove the escape import on line 21 as well

**Comment by anthonydresser** [comment]
I'm not even sure if the original bug is actually a bug. In their exec string they aren't actually using < and > , they are using the entity value for those characters, which is then being escaped since we escape &. If they instead use < and > it would show < and >. This is the same in the query editor.

We are displaying their string correctly as they gave it to us, but they are expecting us to not escape characters, which is not a good approach.

**Comment by alanrenmsft** [comment]
@anthonydresser Thanks for the comments, you are right, it cannot handle html properly, I will look more into it.

**Comment by alanrenmsft** [comment]
The example I used that broke the grid: Exec dbo.ExampleWithXML @XML = '<div style="font-size:15px;">abdcdfd</div>'

I looked further into the problem and found the problem is actually caused by the following code in slick.grid.js

image
it is using the value directly without encoding/escaping it. I can use the custom dataItemColumValueExtractor to provide a object that contains a ariaLabel property but that is not ideal. I think we should fix the issue in slick.grid.js so that we don't have to change all the references of it.

@anthonydresser thoughts?

**Comment by alanrenmsft** [comment]
@anthonydresser I just pushed another commit showing the fixes I had to do within ADS.

File: src/sql/parts/profiler/editor/profilerInput.ts (hunk 1 above)
Comment (alanrenmsft) [reply src/sql/parts/profiler/editor/profilerInput.ts]: thanks! removed the import

**Pull request merged**